core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,39 @@
+const puppeteer = require("puppeteer");
+const basicFunctions = require("./utility-functions/basicFunctions");
+
+const MainDashboard = ".oppia-learner-dashboard-main-content";
+const BlogDashboard = "http://localhost:8181/blog-dashboard";
+const signInInput = "input.e2e-test-sign-in-email-input";
+const editBox = "button.e2e-test-blog-post-edit-box";
+
+puppeteer
+  .launch({
+    headless: false,
+    args: ["--start-fullscreen", "--use-fake-ui-for-media-stream"], // giving microphone and other browser permissions
+  })
+  .then(async (browser) => {
+    const page = await browser.newPage();
+    await page.setViewport({ width: 0, height: 0 }); // for maximizing page height-width
```

Comment:
Can we make these fixed dimensions so that the tests are more robust (don't want different results on different machines)?

Reply:
the tests will run in headless mode. this thing is just for the developers reference so that they can debug easily while writing tests. while merging, we can make headless to be true and can simply remove this line

Reply:
Hmm, if that's the case then I would advise making the comment more explicit. Specifically, say why we do this (I don't quite understand your explanation above so you might want to word it another way) and also say that this needs to be removed before merging. That way, other reviewers will understand the context, too.

Reply:
sure, will do that